iis-workflows/pom.xml

```diff
@@ -56,20 +56,15 @@
         <oozie.package.file.name>oozie-package</oozie.package.file.name>
         <!-- notice: sandboxName is generated based on workflow.source.dir property -->
         <workflow.source.dir>src/test/resources/define/path/pointing/to/directory/holding/oozie_app</workflow.source.dir>
```

Comment:
I don't think it makes any sense to provide this artficial value for the workflow.source.dir. The plugin that needs it should fail if the value is not provided in the command line - not defining it should make the trick, shouldn't it?